### PR TITLE
AZ-444 SignalFx wrapper for python

### PR DESCRIPTION
**Pull request by jmapro**
Prepare v1.0.0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,82 @@
...
+|-----------|----|-----------|
+| azure.function.invocations | Counter | Count number of function invocations |
+| azure.function.errors | Counter | Count number of errors from underlying function handler |
+| azure.function.duration | Gauge | Milliseconds in execution time of underlying function handler |
+
+The function wrapper adds the following dimensions to all data points sent to SignalFx:
+
+| Dimension | Description |
+|-----------|-------------|
+| azure_region | Azure region where the function is executed |
```
Comment (BzSpi) [inline README.md:58]: Cloud we add the resource group and subscription as dimension ?

File: README.md [hunk 2]
```diff
@@ -1 +1,82 @@
...
+| azure_resource_name | Name of the function app where the function is running |
+| function_wrapper_version | SignalFx function wrapper qualifier (e.g. signalfx-azurefunction-0.1.0) |
+| is_Azure_Function | Used to differentiate between Azure App Service and Azure Function metrics |
+| metric_source | The literal value of 'azure_function_wrapper' |
+
+### Deployment
+
+Add the following in your requirements.txt
+```
+git+https://github.com/claranet/signalfx-azure-function-python
```
Comment (BzSpi) [inline README.md]:
```suggestion
git+https://github.com/claranet/signalfx-azure-function-python@vX.X.X
```

**Review by BzSpi — COMMENTED** [review]
Is there a way to to distinguish the "function" called ?

File: README.md (hunk 1 above)
Comment (jmapro) [reply README.md:58]: I can retrieve the resource_group from WEBSITE_RESOURCE_GROUP and extract the subscription_id from WEBSITE_OWNER_NAME variable.

Comment (jmapro) [reply README.md:58]: Done.

File: README.md (hunk 2 above)
Comment (jmapro) [reply README.md]: Updated with version and egg informations

**Comment by jmapro** [comment]
> Is there a way to to distinguish the "function" called ?

Yes, the function_name is retrieved from the context
```   
 default_dimensions["azure_function_name"] = context.function_name
```

**Review by xp-1000 — APPROVED** [review]
very good thanks!

**Pull request merged**